python/coinbase-agentkit/coinbase_agentkit/wallet_providers/eth_account_wallet_provider.py

```diff
@@ -1,49 +1,56 @@
 from decimal import Decimal
 from typing import Any
 
-from eth_account import Account
 from eth_account.datastructures import SignedTransaction
 from eth_account.messages import encode_defunct
+from eth_account.signers.local import LocalAccount
 from pydantic import BaseModel
 from web3 import Web3
 from web3.middleware import SignAndSendRawMiddlewareBuilder
 from web3.types import BlockIdentifier, ChecksumAddress, HexStr, TxParams
 
-from ..network import Network
+from ..network import NETWORK_ID_TO_CHAIN, NETWORK_ID_TO_CHAIN_ID, Network
 from .evm_wallet_provider import EvmWalletProvider
 
 
 class EthAccountWalletProviderConfig(BaseModel):
     """Configuration for EthAccountWalletProvider."""
 
-    private_key: str
-    rpc_url: str
-    chain_id: int
+    account: LocalAccount
+    network_id: str
```

Comment:
I don't think `network_id` is necessary. Can we remove the `EthAccountWalletproviderConfig` object entirely and just pass the `Account` to the ctor. It also doesn't necessarily have to be a local account @0xRAG

Reply:
sure, but should we maybe keep the config object in case the config evolves over time and we do need to end up adding more props? or do you not foresee that happening at all

Reply:
yes makes sense. lets just remove network ID

Reply:
Also, isn't `network_id` necessary? To know which RPC URL to use etc